Core/Sources/Model/LocationUpdate.swift

```diff
@@ -3,15 +3,15 @@ import CoreLocation
 /**
  Enumeration used to determine enhanced location type.
  */
-public enum LocationUpdateType: String, Codable {
+enum LocationUpdateType: String, Codable {
     case predicted = "PREDICTED"
     case actual = "ACTUAL"
 }
 
 /**
  Model used to handle location updates.
  */
-public class EnhancedLocationUpdate {
+class EnhancedLocationUpdate {
```

Comment:
This should still be `public` because it's a class that's exposed in the API :wink: Only the classes that are used for Publisher-Subscriber communication via Ably channels should be private. I'm a bit surprised that after making it invisible outside the SDK the checks have passed :thinking: Aren't you using it in your example apps code?

Reply:
I checked, and the `EnhancedLocationUpdate` and `EnhancedLocationUpdate` is not being used in the example apps.

Okay I think the main issue with setting these public/ not-public is I don't know what the public API is, and am only guessing (quite badly 😅)